constrain/library/zone_temp_control.py

```diff
@@ -5,4 +5,6 @@ class ZoneTempControl(RuleCheckBase):
     points = ["T_set_cool", "T_set_heat"]
 
     def verify(self):
-        self.result = (self.df["T_set_cool"] - self.df["T_set_heat"]) > 2.77
+        self.result = (self.df["T_set_cool"] - self.df["T_set_heat"]) > (
+            2.77 - self.get_tolerance("temperature", "zone")
```

Comment:
This appears to be correct, however, the tolerance has to be specified in deg. C, otherwise it completely throws off the verification. The default tolerance for zone temperature is 0.5 deg. C. Perhaps it should be lower?